src/gettingstarted/local-development/download-code.md

```diff
@@ -0,0 +1,33 @@
+# Local development
+
+## Download the code
+
+If you have already [pushed your code](/gettingstarted/own-code.md) to Platform.sh, then you should already have a local repository that you can build from.
+
+Otherwise, it will be necessary to download a local copy of your project first.
+
+<asciinema-player src="/videos/asciinema/local-copy.cast" preload=1 autoplay=1 loop=1></asciinema-player>
```

Comment:
The end of this video is rather weird.  I don't know what it's doing, but since it loops I can't even see it do it.

Reply:
Took out the loop and autoplay so it should be clearer now once I push a block of edits.